utils/gen_onnx_mlir.py

```diff
@@ -447,6 +447,26 @@
         resultTypes.push_back(attr.cast<TypedAttr>().getType());
       } else if (auto attr = getSparseValueAttr()) {
         resultTypes.push_back(attr.cast<TypedAttr>().getType());
+      } else if (auto attr = getValueFloatAttr()) {
+        resultTypes.push_back(attr.cast<TypedAttr>().getType());
```

Comment:
ONNXConstantOp must have tensor type, so I think you should create a scalar tensor and, moreover, the type should be f32 as for value_floats below
```
RankedTensorType::get({}, FloatType::getF32(attr.getContext()))
```

something similar applies to value_int and value_string

Reply:
Agree.